src/main/java/com/systemmeltdown/robotlib/subsystems/drive/SingleSpeedTalonDriveSubsystem.java

```diff
@@ -1,19 +1,20 @@
 package com.systemmeltdown.robotlib.subsystems.drive;
 
-import java.util.Map;
-
 import com.ctre.phoenix.motorcontrol.ControlMode;
 
 public class SingleSpeedTalonDriveSubsystem extends SkidSteerDriveSubsystem {
-    /**
-	 * Whether or not the left talon group needs to be inverted Value: boolean
-	 */
-    public static final String CONFIG_IS_LEFT_INVERTED = "is_left_inverted";
-    
-    /**
-	 * Whether or not the right talon group needs to be inverted Value: boolean
-	 */
-	public static final String CONFIG_IS_RIGHT_INVERTED = "is_right_inverted";
+
+    public static class Configuration extends SkidSteerDriveSubsystem.Configuration {
```

Comment:
Does this need a constructor?

Reply:
I don't think it does, Stick. There's no defined constructor with any parameters.